routes/service/[service]/component/[component]/[operationType]/op/[operationName]/pipeline/[id]/pause.tsx

```diff
@@ -0,0 +1,50 @@
+import {Handlers, PageProps} from "$fresh/src/server/types.ts";
+import {getServiceMap, pausePipeline} from "../../../../../../../../../../lib/fetch.ts";
+import {GetServiceMapResponse} from "snitch-protos/protos/external.ts";
+import {SuccessType} from "../../../../../../../../../pipelines/[id]/delete.tsx";
+import {Layout} from "../../../../../../../../../../components/layout.tsx";
+import {
+    ReactFlowProvider
+} from "https://esm.sh/v128/@reactflow/core@11.7.4/X-YS9AdHlwZXMvcmVhY3Q6cHJlYWN0L2NvbXBhdCxyZWFjdC1kb206cHJlYWN0L2NvbXBhdCxyZWFjdDpwcmVhY3QvY29tcGF0CmUvcHJlYWN0L2NvbXBhdA/denonext/core.mjs";
+import Flow from "../../../../../../../../../../islands/flow.tsx";
+import {PauseModal} from "../../../../../../../../../../components/modals/pauseModal.tsx";
+import {ResponseCode} from "snitch-protos/protos/common.ts";
+
+
+export const handler: Handlers<any> = {
+    async GET(_req, ctx) {
+        return ctx.render({
+            serviceMap: await getServiceMap(),
+        });
+    },
+    async POST(req, ctx) {
+        const response = await pausePipeline(ctx.params.id);
+        return new Response("", {
+            status: 307,
+            success: {
```

Comment:
I now have a pattern for flashing success messages in session context and showing them after a redirect. See my open PR.